core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -738,6 +768,10 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
         }
     }
 
+    onSeoMediaChange(seoMedia: string) {
+        this.seoMedia = seoMedia;
```

Comment:
Why this is not going in the store? if is changing the UI.

Reply:
Now is in the store